package.json

```diff
@@ -55,5 +58,10 @@
       "vue-cli-service lint",
       "git add"
     ]
+  },
+  "husky": {
+    "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
```

Comment:
can you also add `yarn test:unit` to husky setup?

Reply:
done